fix(curriculum): use example in Modifying an Object Within a Nested Object challenge (#38468)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/modify-an-object-nested-within-an-object.english.md

    online: 80,
    onlineStatus: {
      active: 67,
      away: 13,
      busy: 8
    }
  }
};
```

> <code>nestedObject</code> has three unique keys: <code>id</code>, whose value is a number, <code>date</code> whose value is a string, and <code>data</code>, whose value is an object which has yet another object nested within it. While structures can quickly become complex, we can still use the same notations to access the information we need. To set the value of the <code>busy</code> key to <code>10</code>, we can do so using dot notation by doing:

Reviewer: ```suggestion
<code>nestedObject</code> has three properties: <code>id</code> (value is a number), <code>date</code>(value is a string), and <code>data</code> (value is an object with its nested structure). While structures can quickly become complex, we can still use the same notations to access the information we need. To assign the value <code>10</code> to the <code>busy</code> property of the nested <code>onlineStatus</code> object, we use dot notation to reference the property:
```

> <code>nestedObject</code> has three properties: <code>id</code> (value is a number), <code>date</code>(value is a string), and <code>data</code> (value is an object with its nested structure). While structures can quickly become complex, we can still use the same notations to access the information we need. To assign the value <code>10</code> to the <code>busy</code> property of the nested <code>onlineStatus</code> object, we use dot notation to reference the property:

Reviewer 2: ```suggestion
<code>nestedObject</code> has three properties: <code>id</code> (value is a number), <code>date</code> (value is a string), and <code>data</code> (value is an object with its nested structure). While structures can quickly become complex, we can still use the same notations to access the information we need. To assign the value <code>10</code> to the <code>busy</code> property of the nested <code>onlineStatus</code> object, we use dot notation to reference the property:
```

```js
nestedObject.data.onlineStatus.busy = 10;
```

</section>

## Instructions
<section id='instructions'>

> Here we've defined an object, <code>userActivity</code>, which includes another object nested within it. Set the value of the <code>online</code> key to <code>45</code>.

Reviewer: ```suggestion
Here we've defined an object <code>userActivity</code> that includes another object nested within it. Set the value of the <code>online</code> key to <code>45</code>.
```

> Here we've defined an object <code>userActivity</code> that includes another object nested within it. Set the value of the <code>online</code> key to <code>45</code>.

Reviewer 2: ```suggestion
Here we've defined an object <code>userActivity</code>, which includes another object nested within it. Set the value of the <code>online</code> key to <code>45</code>.
```
